jwst/datamodels/schemas/dark.schema.yaml

```diff
@@ -1,30 +1,28 @@
 title: Dark current data model
 allOf:
 - $ref: referencefile.schema.yaml
-- $ref: instrument_name.schema.yaml
-- $ref: detector.schema.yaml
-- $ref: filter_core.schema.yaml
-- $ref: exposure_type.schema.yaml
+- $ref: keyword_filter.schema.yaml
+- $ref: keyword_exptype.schema.yaml
 - $ref: subarray.schema.yaml
 - type: object
   properties:
     data:
       title: Dark current array
       fits_hdu: SCI
       default: 0.0
-      ndim: 4
+      ndim: 3
       datatype: float32
     dq:
       title: 2-D data quality array for all planes
       fits_hdu: DQ
       default: 0
-      ndim: 4
+      ndim: 3
```

Comment:
I think the original schema had the DQ array defined as 2D. Only the SCI and ERR are 3D.